templates/common/_base/files/usr-local-sbin-set-valid-hostname.yaml

```diff
@@ -0,0 +1,83 @@
+mode: 0755
```

Comment:
You need a filesystem here, it's reqd in ign spec2

Reply:
fixed :)